benchmarks/cpp/nvfuser/utils.cpp

```diff
@@ -142,7 +142,11 @@ void runBenchmarkIterations(
     std::vector<c10::IValue>& aten_inputs) {
   fusion_executor_cache->runFusionWithInputs(aten_inputs);
   bool segmented =
```

Comment:
@shmsong I think you commented on something similar to this, thought I'd point this out too.